### PR TITLE
Lps 186695 | Add functional test for LPS-164912 import Liferay entity should be able to partially updated

### PR DESCRIPTION
**Pull request by yanan**
**Background:**
- Add functional tests on import liferay entities should be able to partially updated

**Test Plan:**
- Given a blog posting is created
And Given new import file in import/export center
And Given "Stop the import on error" unchecked with Only Add New Records and valid JSON file with blog posting entry with existing externalReferenceCode
When importing file with Next button
Then blog posting is not updated
And Then new blog posting entry is created
- Given an account without description field value is created
And Given new import file in import/export center
And Given Add or Update records with Update Changed Record Fields selectd include uploading valid JSON file with 3 accounts including existing one with only required fields
When importing file with Next button
Then existing account is updated the description field value not emptied
And Then new account is created
- Given an account with description field value is created
And Given new import file in import/export center
And Given Add or Update records and Overwrite Records by default selected with uploading valid JSON file with 2 accounts including existing one with only required fields
When importing file with Next button
Then existing account is updated with the description field empty
And Then new account is created

**Jira Ticket:**
- https://issues.liferay.com/browse/LPS-186695

### CODE REVIEW

**Comment by yanan** [comment]
ci:test:headless-functional:nocompile

**Comment by yanan** [comment]
ci:test:sf

**Comment by yanan** [comment]
Functional tests failures are not related.

**Review by Hijky — APPROVED** [review]

File: modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro [hunk 1]
```diff
@@ -449,6 +445,14 @@ definition {
 			value1 = ${strategyType});
 	}
 
+	macro startImportAndWaitForSuccess {
```
Comment (magjed4289) [inline modules/apps/batch-planner/batch-planner-test/src/testFunctional/macros/ImportExport.macro:448]: nice! 👏🏻

**Review by magjed4289 — APPROVED** [review]
looks good!

**Comment by yanan** [comment]
ci:forward:force

**Comment by yanan** [comment]
ci:forward:force

**Pull request closed without merging**